qutip/visualization.py

```diff
@@ -1002,30 +1038,24 @@ def plot_fock_distribution(rho, offset=0, fig=None, ax=None,
 
     ax.bar(np.arange(offset, offset + N), np.real(rho.diag()),
            color="green", alpha=0.6, width=0.8)
+
     if unit_y_range:
         ax.set_ylim(0, 1)
-
     ax.set_xlim(-.5 + offset, N + offset)
     ax.set_xlabel('Fock number', fontsize=12)
     ax.set_ylabel('Occupation probability', fontsize=12)
 
-    if title:
-        ax.set_title(title)
-
     return fig, ax
 
 
 def fock_distribution(rho, offset=0, fig=None, ax=None,
                       figsize=(8, 6), title=None, unit_y_range=True):
     warnings.warn("Deprecated: Use plot_fock_distribution")
-    return plot_fock_distribution(rho, offset=offset, fig=fig, ax=ax,
-                                  figsize=figsize, title=title,
-                                  unit_y_range=unit_y_range)
+    return plot_fock_distribution(rho, offset=offset, fig=fig, ax=ax)
 
 
-def plot_wigner(rho, fig=None, ax=None, figsize=(6, 6),
-                cmap=None, alpha_max=7.5, colorbar=False,
-                method='clenshaw', projection='2d'):
+def plot_wigner(rho, alpha_max=7.5, method='clenshaw', projection='2d', *,
```

Comment:
Could use an input for the smoothness.
The `200` in `xvec = np.linspace(-alpha_max, alpha_max, 200)` could be `N`.

Reply:
Hi. In that case, deleting alpha and adding xvec and yvec (default is None or np.linspace(-7.5, 7.5, 200)) may be better. What do you think of this idea?

Reply:
yes this would be fine.

Reply:
Here is a sample use.
<img width="528" alt="wigner" src="https://github.com/qutip/qutip/assets/72233550/18921533-85cf-4803-a21e-e416e4414e41">